python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py

```diff
@@ -45,7 +47,10 @@ def __init__(self,
         :param output_dim: Integer. Dimension of the dense embedding.
         :param embeddings_initializer: Initializer for the `embeddings`
             matrix (see `keras.initializers`).
-        :param regularizer: Regularizer function applied to
+        :param embeddings_regularizer: Regularizer function applied to
+            the `embeddings` matrix (see `keras.regularizers`).
+            We recommend you not using this parameter.
```

Comment:
"Applying regularizer directly on embeddings will make the sparse gradient dense and may result in degraded performance. We recommend you to use activity_regularizer."